_posts/2020-12-10-share-your-knowledge.markdown

```diff
@@ -0,0 +1,59 @@
+
+One trick I like to pull with Knowledge Shares is to use them to encourage the team's understanding that _learning_ is our paramount goal. That is to say, our goal isn't to _build_ some new feature, but is rather to _learn **how** to build_ that new feature. That's why I emphasize in our Knowledge Shares that _learning is contributing_ (language which I added to our engineering onboarding docs). New team members might not have topics of their own, but they're encouraged to contribute by learning and by asking questions.
+
+When I described the origins of Knowledge Shares above, I said that they worked well on the early Auctions team because the team already had a high level of interpersonal trust. But looking back at those early days, I have to admit that I'm not sure which came first: the Knowledge Shares, or the trust? Having now bootstrapped two product teams at Artsy, I can tell you that Knowledge Shares are a key tool I use as Tech Lead to get teams working well together, quickly. I can't recommend them enough.
+
+Today, Knowledge Shares at Artsy provide a variety of benefits. Among others, those benefits are:
+
+- De-siloing information, to prevent any one engineer from becoming a single-point-of-failure for Artsy's systems.
+- Getting the team used to working together, used to communicating with one another, used to providing and accepting feedback.
+- Surfacing problems developers commonly run into during day-to-day work – and the fixing them permanently, together.
+- Fostering a sense that everyone "owns" this meeting (and, by extension, everyone "owns" the team's culture).
+
+Okay, so, let's imagine I've sold you on the idea of Knowledge Shares. Great! So where do you start? How do you get your team on board? 
+
+Ideally, you _are_ the team's lead, in which case your job is quite easy: send your team this blog post, tell them you want to try Knowledge Shares, try them out, and discuss what everyone things in an upcoming retrospective. Whether you make the Knowledges Shares mandatory, or just encourage everyone's participation, is up to you. They are mandatory for engineers on my team, but I accept the responsibility of making sure the meetings providing value to all engineers. I trust you to make these work well for however your team works.
+
+What if you're _not_ the team lead? If you're comfortable, I would recommend following the same steps as above. Send your team this blog post, express interest in trying out a weekly Knowledge Share, and schedule something. By showing this kind of leadership initiative, it won't be long before you find yourself leading your own team! But if you want to avoid stepping on toes, you can also discuss this with your team's lead privately. Every team is different and I trust you to navigate your own team best.
```

Comment:
Totally non-blocking and kind of rambling:

I was slightly turned off by the phrasing of "Ideally, you _are_ the team's lead" - it triggered some shame in me. 

Thinking about how to rephrase it, I started to realize that I didn't agree with the premise that the team lead is the person that _should_ be responsible/capable of introducing Knowledge Shares (I saw later that you talked about showing leadership initiative, but not before I fought the shame monster). 

This probably isn't the kind of feedback you're looking for, so please feel free to disregard. If it inspires you to reconsider these closing paragraphs though, I do think there's room here to push harder on readers to lead even if "lead" isn't in their title.

Reply:
This is good feedback. I always think about how readers will feel after reading my posts and this feedback is really helpful 👍 You gave me the benefit of the doubt but others won't!